js/repl/Worker.js

```diff
@@ -23,6 +23,13 @@ registerPromiseWorker(message => {
         return null;
       }
 
+    case "getAvailablePresets":
+      try {
```

Comment:
why can it throw?

Reply:
+1

Are you guarding against `Babel` being `undefined` (ie. Babel hasn't loaded yet)? You should probably just check `!!Babel` in that case.